### ISSUE TITLE
VuePdf.createLoadingTask is not a function on 1.9.1

### ISSUE THREAD

**Issue by mesqueeb** [arkokoley/pdfvuer#100]
I previously had this logic:
```js
import VuePdf from "pdfvuer"

// ...

async calculatePdfPageNumber() {
            const { resources } = this
            const [resource] = resources
            if (!resource || resource?.type !== "pdf" || !resource.url) return
            const pdfdata = await VuePdf.createLoadingTask(resource.url)
            this.pageCount = pdfdata.numPages
        },
```
But now that "main" is a Vue file, I'm not able to import this function `VuePdf.createLoadingTask`.
Any advice on how to best approach this?

**Comment by mesqueeb** [comment]
I advice you point "main" towards a file like so:

```js
export { VuePdf, createLoadingTask }
```

that way we can import either the helper function or the Vue Component:

```js
import { VuePdf } from 'pdfvuer'
```

If you want I can make a PR that does this. It would be breaking though : S

**Comment by arkokoley** [comment]
Can you try v1.9.2. This should still work.

**Comment by mesqueeb** [comment]
@arkokoley fixed for me on 1.9.2!
